tls/s2n_handshake_io.c

```diff
@@ -87,7 +87,7 @@ static message_type_t handshakes[64][16] = {
     [INITIAL] = 
     {CLIENT_HELLO, SERVER_HELLO},
 
-    [NEGOTIATED | RESUME ] =
```

Comment:
should we remove the definition of RESUME from tls/s2n_handshake.h if it is no longer used?